internal/commands/result.go

```diff
@@ -814,14 +846,22 @@ func countResult(summary *wrappers.ResultSummary, result *wrappers.ScanResult) {
 		} else if engineType == commonParams.KicsType {
 			summary.KicsIssues++
 			summary.TotalIssues++
+		} else if engineType == commonParams.ContainersType {
+			if wrappers.IsContainersEnabled {
+				*summary.ContainersIssues++
+				summary.TotalIssues++
+			} else {
+				return
```

Comment:
should we return here, or just do nothing?

Reply:
it seems like we shoudn't, make sure its not a bug

Reply:
If Containers are not Enabled- I don't want to summarize it not in the total issues and not in the high\medium\low issues